README.md

```diff
@@ -138,7 +138,7 @@ Run cjdroute without options for HELP:
 
     ./cjdroute
 
-### 0. Make sure you've got the stuff.
+### 0. Make sure you've the stuff.
```

Comment:
`you've the stuff` is not correct, `you have the stuff` is but if you contract `you have` to `you've` you need to add got or something similar.